docs/Agent/config/config.md

```diff
@@ -0,0 +1,22 @@
+### Configuration
+
+**Note**: This Documentation is only required for administrative purposes.
+By Default the collected results are moved to pbench-server.
```

Comment:
When you write "pbench-server" and "pbench-agent", it looks like you're referring to specific commands. Let's stick with "Pbench Agent" and "Pbench Server" to denote the "entities" and avoid that ambiguity. We should also acknowledge (though probably not in much detail here) that one can do a lot with the Pbench Agent command set without ever involving a Pbench Server: pushing datasets to a server is *not* the primary or only purpose of the Pbench Agent, and "by default" is a bit of a stretch considering you need to use a special command and have correctly configured your security settings to make it work.